liquibase-core/src/main/java/liquibase/diff/output/changelog/DiffToChangeLog.java

```diff
@@ -245,12 +245,8 @@ public int compare(DatabaseObject o1, DatabaseObject o2) {
                     if (i != 0) {
                         return i;
                     }
-                } else if (o1 instanceof StoredDatabaseLogic && o1.getAttribute("order", Integer.class) != null
```

Comment:
Not sure that returning here 1 to keep ordering form the LinkedHashSet is a good idea - we try to break design by doing it, which means that we try to go in a wrong way